packages/cluster-ui/src/sessions/sessionsDetailsConnected.stories.tsx

```diff
@@ -40,7 +40,13 @@ const store: Store<AppState> = createStore(
   ),
 );
 
-sagaMiddleware.run(sagas);
+sagaMiddleware.run(
+  initClusterUISagas({
```

Comment:
Why not pass the `SagasContext` as the second param to `sagaMiddleware` and have it be an argument to `sagas` function after the cache invalidation period? That seems like a more straightforward use of the API

Reply:
cache invalidation period is an optional param.  we can't have 2 optional params in a row

Reply:
Gotcha. Can we just have a single configuration param with two optional fields? I don't want to introduce complex `bind` usage and have params come from two different places. It makes the code difficult to read.